app/controllers/main_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MainController < ApplicationController
```

Comment:
It doesn't seem like we're actually using this, so can we delete it?

Reply:
This is the controller that I am using for the search.